app/columns.tsx

```diff
@@ -218,19 +220,11 @@ export const experimentColumns: ColumnDef<ExperimentAndBranchInfo>[] = [
         );
       }
 
-      return ( <></> );
-
-      if (props.row.original.surface !== 'infobar'
-          && props.row.original.surface !== 'spotlight') {
-          return ( <></> );
-      }
-    }
-  }, {
-    accessorKey: "previewLink",
-    header: "",
-    cell: (props: any) => {
       if (props.row.original.previewLink == undefined) {
-        return ( <div/> );
+        const branchLink = getBranchLink(props.row.original.recipe, props.row.original.slug);
+        return (
+          OffsiteLink(branchLink, "JSON/Screenshots")
+        )
```

Comment:
How do we want to combine this with the actual preview link? I'd say it's equally valid to do two different columns (We may want to link to the JSON even if a preview is available) or to only show the screenshots link if the preview isn't present.

Reply:
Those are good questions that I hadn't really thought about.  How about we wait to see what feedback from users looks like (e.g., if they say they miss one when it's not there or ask to see both)?

Reply:
(There is, after all, a complexity tradeoff to showing more affordances on the screen at once).